frontend/src/components/EmployeeModal/EmployeeInfo.tsx

```diff
@@ -1,45 +1,58 @@
 import React from 'react';
 import cn from 'classnames';
-import { useFormContext } from 'react-hook-form';
+import {useFormContext } from 'react-hook-form';
 import styles from './employeemodal.module.css';
 import { Input } from '../FormElements/FormElements';
 
 
 const EmployeeInfo = () => {
-  const { register } = useFormContext();
+  const { register, formState } = useFormContext();
+  const {errors} = formState;
   return (
     <div className={styles.inputContainer}>
-      <Input
-        name='name'
-        type='text'
-        placeholder='Name'
-        className={cn(styles.input, styles.col1)}
-        ref={register({ required: true })}
-      />
-      <Input
-        name='netid'
-        type='text'
-        placeholder='NetID'
-        className={cn(styles.input, styles.col2)}
-        ref={register({ required: true })}
-      />
-      <Input
-        name='email'
-        type='email'
-        placeholder='Email'
-        className={cn(styles.input, styles.col1)}
-        ref={register({ required: true })}
-      />
-      <Input
-        name='phoneNumber'
-        type='tel'
-        pattern="[0-9]{10}"
-        placeholder='Phone Number'
-        min={10}
-        max={10}
-        className={cn(styles.input, styles.col2)}
-        ref={register({ required: true })}
-      />
+      <div className = {styles.col1}>
+        <Input
+          name='name'
+          type='text'
+          placeholder='Name'
+          className={cn(styles.input)}
+          ref={register({ required: "Please enter a valid name" })}
+        />
+        {errors.name && <p className={styles.error}>{errors.name.message}</p>}
+      </div>
+      <div className = {styles.col2}>
+        <Input
+          name='netid'
+          type='text'
+          placeholder='NetID'
+          className={cn(styles.input)}
+          ref={register({ required: "Please enter a valid NetID" })}
+        />
+        {errors.netid && <p className={styles.error}>{errors.netid.message}</p>}
+      </div>
```

Comment:
![image](https://user-images.githubusercontent.com/52215742/114112578-a5e45900-98aa-11eb-8bc9-9f549435b12f.png)
![image](https://user-images.githubusercontent.com/52215742/114112645-c4e2eb00-98aa-11eb-8f47-c05a6374a781.png)

There are some weird artifacts with the input sizing (maybe show some borders to see where the issue is). I think the email input is too long.


I'm not sure if this is the intended design, but the contrast is bad for the day when I click on it.
![image](https://user-images.githubusercontent.com/52215742/114112696-e5ab4080-98aa-11eb-8ef4-c6a2b1905399.png)

It probably has to do with focus coloring since when I click elsewhere I get.
![image](https://user-images.githubusercontent.com/52215742/114112739-feb3f180-98aa-11eb-841a-ee69ea5884b3.png)